mautic/mautic
# [UX] Change Filters message on New Segment

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

The current message displayed when setting filters on a New Segment could be streamlined for clarity. It would be beneficial to simplify the language and make it more concise. The proposed solution in this pull request is to change the message to “Contacts that match the filters will be added, and contacts that no longer match will be removed. Those manually added will remain untouched.”

This change is beneficial for users who are creating new segments and could benefit from clearer instructions. By implementing this change, we can expect a more user-friendly experience when creating new segments, potentially making the platform more accessible to a wider range of users. This could lead to users being more comfortable with using the platform and exploring its features.

![image](https://github.com/mautic/mautic/assets/116097999/6a5f3617-0d99-4ea4-8a48-f153957f7378)







#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments page
3. Create a new segment
4. Go to the Filters tab
5. See the new message in blue
